src/data/knowledge_graph.csv

```diff
@@ -680,23 +680,117 @@ AR_Z_049,Q615066,
 AS,Q16641,country/ASM
 AT,Q40,country/AUT
 AT_1,Q43210,ISO/AT-1
+AT_1_101,Q126321,wikidataId/Q126321
```

Comment:
If we know the NUTS code, we prefer to use `nuts/${code}` (even if the entry does not exist in DataCommons yet)

Reply:
We know the NUTS codes for the six districts that correspond precisely to NUTS regions.  Should I change those to use nuts/${code} and use wikidataId/${code} for the others?

Reply:
Any particular reason why we don't want to include the wikidataId/${code} for the districts that are not NUTS?

Reply:
wikidataId does show up in datacommons, e.g., https://datacommons.org/place/wikidataId/Q13298